src/ui/windows/TogglDesktop/TogglDesktop/ui/views/Timeline.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<UserControl x:Class="TogglDesktop.Timeline"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls"
+             xmlns:viewModels="clr-namespace:TogglDesktop.ViewModels"
+             xmlns:togglDesktop="clr-namespace:TogglDesktop"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+             d:DataContext="{d:DesignInstance viewModels:TimelineViewModel, IsDesignTimeCreatable=False}">
+    <Grid Background="{DynamicResource Toggl.Background}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"></RowDefinition>
+            <RowDefinition Height="Auto"></RowDefinition>
+            <RowDefinition Height="*"></RowDefinition>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto"></ColumnDefinition>
+            <ColumnDefinition Width="*"></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+        <mah:ToggleSwitch Margin="10 14 0 0"
+                          OnLabel=""
+                          OffLabel=""
+                          IsChecked="{Binding RecordActivity}"
+                          Grid.Row="0" Grid.Column="0">
+            Record activity
+        </mah:ToggleSwitch>
+        <Viewbox Width="15" Height="15" Grid.Row="0" Grid.Column="1"
+                 HorizontalAlignment="Left"
+                 VerticalAlignment="Bottom"
+                 Margin="12 0 0 0"
+                 Name="RecordActivityInfoBox"
+                 MouseEnter="RecordActivityInfoBoxOnMouseEnter">
+            <Grid>
+                <Ellipse Fill="{DynamicResource Toggl.DisabledTextBrush}"

```

Comment:
We already have a nice info icon that is used in the Project creation view. See `Toggl.InfoIconGeometry`.
![image](https://user-images.githubusercontent.com/16451391/89434409-2ebf5980-d74c-11ea-9852-01311479732e.png)
P.S. the icon is also present in Zeplin designs of this page, you had to use it if you didn't know about the existing one.